metricbeat/docs/gettingstarted.asciidoc

```diff
@@ -267,13 +280,17 @@ docker run {dockerimage}
 [source,shell]
 ----------------------------------------------------------------------
 sudo chown root metricbeat.yml <1>
-sudo ./metricbeat -e -c metricbeat.yml -d "publish"
+sudo chown root modules.d/system.yml <1>
+sudo ./metricbeat run -e -c metricbeat.yml -d "publish"
 ----------------------------------------------------------------------
 <1> You'll be running Metricbeat as root, so you need to change ownership
-of the configuration file (see
+of the configuration file and any configurations enabled in the `modules.d`
+directory (see
 {libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
 in the _Beats Platform Reference_).
 
+//REVIEWERS: It seems really inconvenient to users to expect them to change ownership on all the files in `modules.d`. This is something I had to do, but not 100% sure if there is a better way. Please advise.
+
```

Comment:
@andrewkroh Ok, I've added the note about `-strict.perms`. I know we're talking about the getting started experience here, but I sort of worry if we start recommending something in the GS that could get people into trouble later on. See how this works. If the developers on the team decide that we want to tell users straight out to run the the beats with `-strict.perms=false` then I can change it, but the way it's written now users at least understand that there's a choice to be made...and might do more reading before they run the command blindly.